prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Anna Aksenov';
```

Comment:
Hey Anna, just update this by changing `var` to `const` 

never should still be defined with `let`